usaspending_api/disaster/v2/views/overview.py

```diff
@@ -24,15 +25,12 @@ class OverviewViewSet(DisasterBase):
     def get(self, request):
 
         request_values = self._parse_and_validate(request.GET)
-        defc = request_values["def_codes"].split(",")
+        self.defc = request_values["def_codes"].split(",")
```

Comment:
Minor, but this could just use self.filters["def_codes"] because it is built off of DisasterBase. Would also mean removing `_parse_and_validate`. Not worth blocking over, but It seems unnecessary